cmd/kube-apiserver/app/server.go

```diff
@@ -733,16 +733,14 @@ func getServiceIPAndRanges(serviceClusterIPRanges string) (net.IP, net.IPNet, ne
 		return apiServerServiceIP, primaryServiceIPRange, net.IPNet{}, nil
 	}
 
-	if len(serviceClusterIPRangeList) > 0 {
-		_, primaryServiceClusterCIDR, err := net.ParseCIDR(serviceClusterIPRangeList[0])
-		if err != nil {
-			return net.IP{}, net.IPNet{}, net.IPNet{}, fmt.Errorf("service-cluster-ip-range[0] is not a valid cidr")
-		}
+	_, primaryServiceClusterCIDR, err := net.ParseCIDR(serviceClusterIPRangeList[0])
```

Comment:
need release notes?

Reply:
Thanks for review！

Given the 'if-condition' in [line 727](https://github.com/kubernetes/kubernetes/blob/4925cb66c942b4b4ce3677d343d08f2c5a393fa2/cmd/kube-apiserver/app/server.go#L727 ) , it's clear enough that we don't need notes.